code/game/objects/items/stacks/packagewrap.dm

```diff
@@ -195,7 +197,11 @@
 
 /obj/item/delivery/large/attack_hand(mob/user as mob)
 	if(wrapped)
-		wrapped.forceMove(get_turf(src.loc))
+		wrapped.forceMove(get_turf(loc))
```

Comment:
`get_turf()` is basically equally fast regardless of nesting depth now
It does slow down marginally with deeper nesting, but it's only measurable on the scale of 10 million calls or more
It might even be slower to read `loc`